src/Simulation/Core/Submitters/IQuantumMachine.cs

```diff
@@ -14,8 +14,13 @@ namespace Microsoft.Quantum.Runtime
     // 
     // TODO: deprecate once IQSharpSubmitter is fully hooked up.
     //[Obsolete("Replaced by IQSharpSubmitter.")]
-    public interface IQuantumMachine : IAzureSubmitter
+    public interface IQuantumMachine
     {
+        /// <summary>
+        /// The name of the execution target.
+        /// </summary>
+        string Target { get; }
+
```

Comment:
I may be wrong but I have an impression that `IQSharpSubmitter`, `IQirSubmitter`, and `IQuantumMachine` can have a common base class/interface with contents:
```c#
        /// <summary>
        /// The name of the execution target.
        /// </summary>
        string Target { get; }
```

Reply:
The plan is to deprecate `IQuantumMachine` and both `IQSharpSubmitter` and `IQirSubmitter` do share `Target` as a member but I think it would be overdoing it just creating an interface with one member in it.